react/finger-exercises/redux/02 - react-redux/src/redux/shoppingCart/actions.js

```diff
@@ -0,0 +1,12 @@
+export const actions = {
+  TOOGLE_CONTENT: '@@SHOPINGCART/TOOGLE_CONTENT'
+};
+
+const actionsCreators = {
+  toggleContent: () => ({
+    type: actions.TOOGLE_CONTENT,
+    open
```

Comment:
As a convention  we follow, actions should be defined with `type` and `payload` properties and `payload` would contain the fields that are going to be updated.
```js
toggleContent: () => ({
    type: actions.TOOGLE_CONTENT,
    payload: { open }
  })
};
```
Also, what is open? Are you missing the param?
In the reducer you just set the value negating the previous one.